services/catalog/src/simcore_service_catalog/services/frontend_services.py

```diff
@@ -25,11 +29,11 @@ def _file_picker_service() -> ServiceDockerData:
     )
 
 
-def _node_group_service() -> ServiceDockerData:
+def create_node_group_service() -> ServiceDockerData:
     return ServiceDockerData(
-        key="simcore/services/frontend/nodes-group",
+        key=f"{FRONTEND_SERVICE_KEY_PREFIX}/nodes-group",
         version="1.0.0",
-        type="frontend",
+        type=ServiceType.FRONTEND,
```

Comment:
why here frontend and not the file-picker? these should be the same.

Reply:
I asked the same question to @odeimaiz. For some reason the file-picker was never set as front-end. Should I set it as front-end?

Reply:
OK, I just checked the frontend code. It doesn't care what type the file-picker is